transforms/code/license_check/README.md

```diff
@@ -0,0 +1,13 @@
+# License Check 
+
+The License Check transform checks if the '`license` of input data is in approved/denied list as 
```

Comment:
This sentence looks incomplete. The transformer verifies and marks the sample accordingly in the output right? should we convey that in this description.